examples/gnrc_border_router/Makefile

```diff
@@ -39,8 +39,9 @@ USEMODULE += ps
 # Optionally include DNS support. This includes resolution of names at an
 # upstream DNS server and the handling of RDNSS options in Router Advertisements
 # to auto-configure that upstream DNS server.
-#USEMODULE += sock_dns              # include DNS client
-#USEMODULE += gnrc_ipv6_nib_dns     # include RDNSS option handling
+USEMODULE += sock_dns              # include DNS client
```

Comment:
```suggestion
USEMODULE += dns_cache             # cache DNS responses
USEMODULE += sock_dns              # include DNS client
```

Reply:
Also, I think here too some `ENABLE_DNS ?= 1` switch or something of the like would be helpful, in case e.g. you don't want the border router include RDNSS options in its RAs (e.g. to prevent RA fragmentation).

Reply:
I tried to address both comments in my latest commit.